src/main/java/org/reaktivity/nukleus/http2/internal/types/stream/Http2ContinuationFW.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright 2016-2017 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.http2.internal.types.stream;
+
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.reaktivity.nukleus.http2.internal.types.Flyweight;
+
+import java.nio.ByteOrder;
+import java.util.function.Consumer;
+
+import static java.nio.ByteOrder.BIG_ENDIAN;
+import static org.reaktivity.nukleus.http2.internal.types.stream.Http2Flags.END_HEADERS;
+import static org.reaktivity.nukleus.http2.internal.types.stream.Http2FrameType.CONTINUATION;
+
+/*
+
+    Flyweight for HTTP2 CONTINUATION frame
+
+    +-----------------------------------------------+
+    |                 Length (24)                   |
+    +---------------+---------------+---------------+
+    |   Type (8)    |   Flags (8)   |
+    +-+-------------+---------------+-------------------------------+
+    |R|                 Stream Identifier (31)                      |
+    +=+=============+===============================================+
+    |                   Header Block Fragment (*)                 ...
+    +---------------------------------------------------------------+
+
+ */
+public class Http2ContinuationFW extends Flyweight
```

Comment:
Removed in all flyweight classes, except for Http2DataFW, Http2FrameFW as they conflict with the generated ones.